package-lock.json

```diff
@@ -2697,9 +2697,9 @@
       }
     },
     "node_modules/codemirror": {
-      "version": "5.65.4",
```

Comment:
Shouldn't the `package.json` be updated as well?

Reply:
No, we don't directly depend on it.